docs/guides/getting-started-java-spring.md

```diff
@@ -125,7 +126,8 @@ Add the following Maven dependency to your Spring Boot Starter project, replacin
 
 ### Configure the Camunda client
 
-Open your `src/main/resources/application.yaml` file, and paste the following snippet to connect to the Self-Managed Zeebe Broker:
+1. [Rename your `src/main/resources/application.properties` file to `application.yaml`](https://mvnrepository.com/artifact/io.camunda/spring-boot-starter-camunda-sdk).
```

Comment:
The original download from start.spring.io is in the plain-text format (application.properties), so if we want them to configure via yaml, we should tell them to change that file type.